docs/features/Installation.md

```diff
@@ -75,14 +89,24 @@ In your linux environment create a folder for the gateway ( `cd /var/www/gateway
 Then run either `$ composer require common-gateway/core-bundle` or the composer require command for a specific plugin.
 
 
+## Troubleshooting
+During installation, it is possible that you run into problems, below you will find common problems and how to deal with them. If you are still running into problems after reading this or if you have any constructive criticism please seek contact with our development team (info@conduction.nl).
+> Note: when troubleshooting you will, in most cases, need to run some commands. Unless stated otherwise, these commands should always be executed in the php container.
+
+A very common way to check why the Common Ground Gateway is not functioning as expected is by running the `bin/console doctrine:schema:validate` command. This is a command that validates the mappings of the Commonground Gateway database, but will often return insightful error messages when running into other problems as well.
+
+**You have requested a non-existent service**
+
+If you get the error message "You have requested a non-existent service" then this indicates in most cases that your config/bundles.php file isn't up-to-date. This bundles.php file should contain all installed bundles. If you, for example, get a message that `OpenCatalogi\OpenCatalogiBundle\ActionHandler\ComponentenCatalogusApplicationToGatewayHandler` does not exist, you most likely need to add `OpenCatalogi\OpenCatalogiBundle` to the bundles.php file like this: `OpenCatalogi\OpenCatalogiBundle\OpenCatalogiBundle::class => ['all' => true],` locally you can just edit this file, with a server installation you might want to use something like [vi editor](https://www.redhat.com/sysadmin/introduction-vi-editor) for this.
```

Comment:
This off course only applies if the bundle has been installed and you get this error